### DISCUSSION TITLE
Notification about new strings to be translated

### DISCUSSION THREAD

**Discussion by dobridabar** [organicmaps/organicmaps#9933] [Q&A]
Regarding this commit:

https://github.com/organicmaps/organicmaps/commit/464c2454baa15714813b603c005d82d521f894a7

I see that there is a new string [placepage_delete_track_confirmation_alert_message], where most of the languages are added, but Serbian is not.

I suppose that when you add a new string, you notify your translators, and they do the translation for every language. Is this correct assumption? If yes, can I be added to the list?

CC: @biodranik

**Comment by pastk** [comment]
@rtsisyk could you please add @dobridabar to the @organicmaps/translations-sr team

**Reply by dobridabar** [reply]
@rtsisyk Gentle reminder
